amplipi/updater/static/index.html

```diff
@@ -44,7 +44,7 @@ <h1><span class="text-white">Ampli</span><span class="text-danger">Pi</span></h1
               <a class="nav-link" id="manual-update-tab" data-toggle="tab" href="#manual-update" role="tab" aria-controls="manual-update" aria-selected="false">Custom Update</a>
             </li>
             <li class="nav-item">
-              <a class="nav-link" id="admin-settings-tab" data-toggle="tab" href="#admin-settings" role="tab" aria-controls="admin-settings" aria-selected="false">Admin Settings</a>
+              <a class="nav-link" id="admin-settings-tab" data-toggle="tab" href="#admin-settings" role="tab" aria-controls="admin-settings" aria-selected="false" onClick="getPersist();">Admin Settings</a>
```

Comment:
Now that the state of that tab can be changed by updating, change the update method from `window.onload` to `tab.onclick`